pdf-service/package.json

```diff
@@ -43,6 +43,7 @@
     "kafka-node": "^4.1.3",
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2",
+    "moment-timezone": "^0.5.28",
```

Comment:
Correct the `package.json`, currently it is taken from sample project and has wrong project name and urls